addon_service/authorized_storage_account/serializers.py

```diff
@@ -69,12 +38,18 @@ def __init__(self, *args, **kwargs):
         related_link_view_name=view_names.related_view(RESOURCE_TYPE),
         required=False,
     )
+    url = serializers.HyperlinkedIdentityField(
+        view_name=view_names.detail_view(RESOURCE_TYPE), required=False
+    )
+    account_owner = ReadOnlyResourceRelatedField(
+        many=False,
+        queryset=UserReference.objects.all(),
+        related_link_view_name=view_names.related_view(RESOURCE_TYPE),
+    )
```

Comment:
Out of curiosity, what prevented these from being included in the base serializer? Could you add that to the PR description so that we can all learn?

Reply:
`RESOURCE_TYPE` is dependent on dataclass which is dependent on subclass.
Surely, we can abstract this one, but it'd be a lot more messy, and will look as follows:
``` 
def AuthorizedAccountSerilzer(RESOURCE_TYPE: str):
    class _AuthroizedAccountSerilzer(serializers.HyperlinkedModelSerializer):
    ...
     account_owner = ReadOnlyResourceRelatedField(
        many=False,
        queryset=UserReference.objects.all(),
        related_link_view_name=view_names.related_view(RESOURCE_TYPE),
    )
```
and subclass it as follows

```
RESOURCE_TYPE = get_resource_type_from_model(AuthorizedStorageAccount)

class AuthroizedStorageAccountSerializer(AuthorizedAccountSerilzer(RESOURCE_TYPE)):

```

This is more DRY, drastically but more complex, obscure and hard to maintain.